.eslintrc.js

```diff
@@ -120,7 +120,8 @@ module.exports = {
     'AsyncIterable': 'readonly',
     'FileSystemFileEntry': 'readonly',
     'FileSystemDirectoryEntry': 'readonly',
-    'FileSystemEntry': 'readonly'
+    'FileSystemEntry': 'readonly',
+    'IS_REACT_ACT_ENVIRONMENT': 'readonly'
```

Comment:
https://github.com/adobe/react-spectrum/pull/6978/commits/7f57237d7d629ad83a1d9d5e9ad9137c64082952

Didn't think we should have to rewrite tests to account for the delay on first announce and didn't want to disrupt downstream users tests